api/v1alpha2/hierarchicalresourcequota_types.go

```diff
@@ -38,10 +38,16 @@ type HierarchicalResourceQuotaStatus struct {
 	// and its descendant namespaces.
 	// +optional
 	Used corev1.ResourceList `json:"used,omitempty"`
+	// +optional
+	Request string `json:"request,omitempty"`
+	// +optional
+	Limit string `json:"limit,omitempty"`
```

Comment:
Is "Request" just a human-readable version of "Used," and "Limit" a human-readable version of "Hard"?

I'm a bit nervous about this, because I was trying to keep the API for HRQ the same as regular RQ, and [regular RQ doesn't have these fields](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.27/#resourcequotastatus-v1-core). If we need these anyway (e.g. because `kubectl` understands regular RQ but not HRQ), then we should give them more descriptive names IMO along with documentation, e.g. `HardString` and `UsedString` with comments that clearly indicate why they exist.

wdyt?

Reply:
Ah, I see. How about `RequestsSummary` and `LimitsSummary`, and then document them in the comments (e.g., "RequestsSummary is used by `kubectl get hrq`, and summarizes the relevant information from `.status.hard.requests` and `.status.used.requests`".